R/save.r

```diff
@@ -13,6 +13,16 @@
 #' containing the `%` sign, use `%%`. For example, `filename = "figure-100%%.png"`
 #' will produce the filename `figure-100%.png`.
 #'
+#' @section Writing to a graphics device:
+#'
+#' In most cases `ggsave()` is the simplest way to save your plot, but
+#' sometimes it is necessary to save the plot by writing directly to a
+#' graphics device. For example, to create a PNG file, you could first
```

Comment:
I think there's a small break in logic here. "...sometimes it is necessary to save the plot ... For example, ..." At this point I expect an example of *why* it is necessary to save the plot, but instead I'm given an example of how to get this done.

You could either provide an example of why one might want to do this, or write a better transition. To write a better transition, you could add: "In this case, you can just open a regular R graphics device, print the plot, and then close the device. For example, ..."

Reply:
The updated version uses a transition similar to your suggestion, and moves the code illustrating the logic to the examples section. I think this version is clearer than the previous one